eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -63,6 +63,7 @@ steps:
 
   # Attempt to restore vcpkg from the cache
   - task: Cache@2
+    timeoutInMinutes: 1
```

Comment:
What is the default timeout for this task? It is less then 1 min?

Reply:
Looks like it is 30 seconds.